eng/common/pipelines/templates/steps/get-pr-owners.yml

```diff
@@ -1,5 +1,7 @@
-parameters: 
+parameters:
   TargetVariable: ''
```

Comment:
leaft this variable to prevent a break when this PR is merged.

Reply:
Great lets file a follow-up to remove it once we convert all usages.

Reply:
https://github.com/Azure/azure-sdk-tools/issues/1035